### DISCUSSION TITLE
leaderboard scores seem broken

### DISCUSSION THREAD

**Discussion by GopoBoi** [ppy/osu#20058] [Q&A] [answered]
![osu_2022-09-01_10-09-52](https://user-images.githubusercontent.com/69014805/187866791-fa5e129f-f8f8-4f98-b989-1478158d6592.jpg)
I don't think these should be over 1M score with 80% accuracy with standardized scoring,
it looks even more broken with classic score display:
![osu_2022-09-01_10-19-15](https://user-images.githubusercontent.com/69014805/187867123-df3b9e7d-1177-4d9f-a5d8-06da1b390a4b.jpg)

**Comment by peppy** [comment]
Does this only happen on the latest release? Please link the beatmaps you're seeing this on. Please include the version you're running.

**Reply by GopoBoi** [reply]
just checked. doesn't happen on 2022.821.0, only the latest release (2022.901.0). 
On closer look it also looks like all the broken scores have been submitted in the last 3 days.
I'm seeing this on most beatmaps, but here are a few:
https://osu.ppy.sh/beatmapsets/1781620#osu/3659920
https://osu.ppy.sh/beatmapsets/1806908#osu/3705535
https://osu.ppy.sh/beatmapsets/1629837#osu/3339802
https://osu.ppy.sh/beatmapsets/57460#osu/173196

**Reply by Coppertine** [reply]
https://osu.ppy.sh/beatmapsets/594751#osu/1257904
https://osu.ppy.sh/beatmapsets/594751#osu/1257904
These maps as well where people SS a map with only 1-4 objects.
![osu_2022-09-01_22-13-47](https://user-images.githubusercontent.com/37494321/187911471-e2efb6f3-e1bf-4d33-98d7-6dfa6b68b013.png)
![image](https://user-images.githubusercontent.com/37494321/187911262-c0f7a8af-5165-491b-a318-988503638813.png)

**Reply by GopoBoi** [reply]
@Coppertine that is a completely different issue that has already been fixed as far as I know.

**Reply by isuewo** [reply]
I have the same issue with 2022.901.0. I have also created #20068 because I didn't see this discussion, should I close it now?
I can confirm that it looks like all the broken scores have been submitted in the last 3 days. Another thing I noticed is that my scores submitted with 2022.901.0 seem to be fine.

**Comment by peppy** [comment, marked as answer]
A fix is currently being applied to fix this server-side. You should notice things start to improve over the coming hours.
